packages/esm-patient-list-app/src/patient-list-list/patient-list-list.component.tsx

```diff
@@ -99,62 +96,22 @@ const PatientListList: React.FC = () => {
     [toggleStarredMutation, patientListQuery, userId],
   );
 
+  const handleSearch = (str) => setSearchString(str);
+
+  const handleCreateListSuccess = useCallback(() => {
+    patientListQuery.refetch();
+  }, [patientListQuery]);
+
```

Comment:
I think that these redefinitions could also be removed in favor of just calling/passing the wrapped functions directly.

Reply:
The same `useCallback` situation applies here as well. `patientListQuery` is stable and `patientListQuery.refetch` too (though this entire construct should ideally be replaced with SWR anyway).

For the `setSearchString` wrapper, there isn't even a `useCallback` used which causes `handleSearch` to change during each render. Note that this is not the case with the state setter `setSearchString` - using `setSearchString` directly instead of the custom `handleSearch` is definitely better.

Reply:
Got it @manuelroemer ! Thanks alot!